versions.props

```diff
@@ -16,6 +16,7 @@ com.google.guava:guava = 28.0-jre
 com.github.ben-manes.caffeine:caffeine = 2.8.4
 org.apache.arrow:arrow-vector = 5.0.0
 org.apache.arrow:arrow-memory-netty = 5.0.0
+io.netty:netty-buffer = 4.1.63.Final
```

Comment:
+1 to moving the actual version over to this file instead of inline.

Question: Is there anyway to just enforce that it uses the version from `iceberg-arrow` without including this? I worry about having to keep track of such an important dependency.

Reply:
I think it's good to make the version explicit globally, so that people understand the version better inside the shaded jar.